api/src/main/java/org/openmrs/module/appointments/dao/impl/AppointmentDaoImpl.java

```diff
@@ -117,4 +118,13 @@ public Appointment getAppointmentByUuid(String uuid) {
         }
         return criteria.list();
     }
+
+    @Override
+    public List<Appointment> search(AppointmentSearch appointmentSearch) {
+        Criteria criteria = sessionFactory.getCurrentSession().createCriteria(Appointment.class);
+        criteria.add(Restrictions.eq("voided", false));
+        Date maxEndDate = new Date(appointmentSearch.getEndDate().getTime());
```

Comment:
- Any particular reason, for endDate - we call - appointmentSearch.getEndDate().getTime(), 
While for startDate its just appointmentSearch.getStartDate().

Reply:
In the previous commit, we added a millisecond to it. So converted to time and added a millisecond. Now we are doing that at another place but forgot to remove that declaration. Can be removed now